device/globals/global_vm.gd

```diff
@@ -191,6 +191,9 @@ func inventory_set(p_obj, p_has):
 func say(params, level):
 	get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "dialog", "say", params, level)
 
+func set_hud_visible(is_visible):
+	get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "game", "set_hud_visible", is_visible)
```

Comment:
Instead of storing a reference to the hud and calling this additional function, you could just call `get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "game", "menu_closed")` (or "menu_opened") instead? Currently I think only the hud will be triggered by that call, but it could be used by other ui elements which need to close programmatically in the future.

Reply:
The HUD has its own group too, so I think a clean solution would be to add another function, eg. `toggle_visibility(p_visible)` to hud.gd which takes a boolean to show/hide and set the `visible` property when called via group call.